### ISSUE TITLE
Review mouse tp53 annotation to GO:0019661 glucose catabolic process to lactate via pyruvate

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-annotation#5595]
Hi @LiNiMGI 

@cmungall found a potentially incorrect mouse annotation, see https://github.com/geneontology/go-ontology/issues/29465

> for mouse, this publication
https://amigo.geneontology.org/amigo/reference/PMID:20935145
yields an annotation of mouse tp53 to [glucose catabolic process to lactate via pyruvate](https://amigo.geneontology.org/amigo/term/GO:0019661) but it's clearly acting as a regulator

Could you please review it? 

Thanks, Pascale

**Comment by LiNiMGI** [comment]
The noctua model looks right to me...
![Screenshot 2024-12-19 at 12 56 36 PM](https://github.com/user-attachments/assets/6c99c9f6-ceb7-4241-8410-a9f0dc23334a)

**Comment by pgaudet** [comment]
Hi @LiNiMGI 

Thanks for looking into this. In fact we have discussed the guidelines for regulation many times over the years, and my understanding is that we had agreed that we needed to know the molecular mechanism of a regulation to annotate to regulation. 

I see two concerning things in this model: 
1. MF root, ie the molecular mechanism linking Trp53 and the downstream process is unknown
2. The relation 'acts upstream of or within' also suggests that the connection between the annotated gene and the downstream process is unknown. 

So, as this model is right now, I agree with @cmungall the regulation is an over annotation. 

@vanaukenk @ValWood @thomaspd @hattrill This suggests we could implement some guidelines for annotations to BP regulation: 

1. A MF can only be annotated to a regulation BP if its activity directly regulates an activity that is part of that BP
2. A MF cannot be connected to a  regulation BP by 'acts upstream of or within' 

Maybe this would be too stringent? But it would be useful to look at a report of these types of annotations to make sure we're not capturing indirect/upstream effects that are not regulatory. 

Thanks, Pascale

**Comment by ValWood** [comment]
I don't think you can make this type of curation in the pathway editor?

**Comment by pgaudet** [comment]
No

**Comment by LiNiMGI** [comment]
Hi @pgaudet 
I thought as Chris pointed out: it is clearly acting as a regulator, but it is annotated to [glucose catabolic process to lactate via pyruvate](https://amigo.geneontology.org/amigo/term/GO:0019661)

Looking at the noctua model, MGI only annotated to regulation,  but when noctua generate GPAD, it generate both the regulation annotation and a directed annotation to glucose catabolic process to lactate via pyruvate.
Thanks,
Li

**Comment by LiNiMGI** [comment]
@ValWood this is the graph view of the original MGI noctua model.

**Comment by vanaukenk** [comment]
My understanding is that this annotation lives in the gene-centric, imported MGI model for Trp53.

The annotation to the root MF term that is linked to the original Biological Process annotation is just the convention for how standard BP annotations are expressed in the GO-CAM data model and thus, were imported into Noctua.

The original annotation also contained a 'regulates o occurs in' extension and that's why we see the construct we do in the GO-CAM conversion.

MGI:MGI:98834 RO:0002264 GO:1904024 MGI:MGI:4867601|PMID:20935145 ECO:0000315 2015-03-16 MGI BFO:0000066(CL:0000746),GOREL:0001004(CL:0000746) creation-date=2015-03-16|modification-date=2015-03-16|comment=evidence RNAi evidence ; ECO:0000019|contributor-id=https://orcid.org/0000-0001-7476-6306

In the Noctua src GPAD for MGI, I see annotations to 'acts upstream of' for GO:0019661 and an 'acts upstream of or within' annotation to GO:1904024, so I suspect there may be a property chain issue here that needs checking wrt the Noctua GPAD output.  Unfortunately, we can't see the gp2term relations in AmiGO.

@pgaudet is correct that we don't want to annotate to regulation if we don't know the mechanism, but I don't _think_ that's exactly what's happening here.

We need to prioritize working on the Noctua GPAD output in the new year.  The Annotation Preview display in Noctua looks to be showing the annotation to the regulation term with the 'acts upstream of or within' qualifier which is what I would have expected to see in the GPAD output.

**Comment by ukemi** [comment]
> We need to prioritize working on the Noctua GPAD output in the new year.

Yes!!!! But in this case, I would argue that the GPAD is correct with an 'acts upstream of' qualifier based on the original annotation.  @vanaukenk is correct that there is a loss of information going from the original annotation that was imported into Noctua and the lossy final representation of the data. I think the decision here is whether or not to include genes that definitely act upstream of a process in a final flat annotation file, especially if no one is going to represent or filter GP2term relationships. This is more a GPAD issue and a decision about what qualifiers are going to be allowed in the file.
